test/testdata/java_shelf_book_name.baseline

```diff
@@ -27,13 +27,18 @@ import java.util.List;
 public class ShelfBookName extends BookName {
 
   private static final PathTemplate PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding("shelves/{shelf}/books/{book}");
+      PathTemplate.createWithoutUrlEncoding("projects/{project}/shelves/{shelf}/books/{book}");
```

Comment:
All the changes in this file result from the path "shelves/{shelf}/books/{book}" being changed to "projects/{project}/shelves/{shelf}/books/{book}"